presto-main/src/main/java/io/prestosql/execution/scheduler/SqlQueryScheduler.java

```diff
@@ -474,6 +474,13 @@ public StageInfo getStageInfo()
         return buildStageInfo(rootStageId, stageInfos);
     }
 
+    public List<StageInfo> getAllStages()
```

Comment:
There already is `io.prestosql.execution.StageInfo#getAllStages`, so in `SqlQueryExecution` you could use:
```
io.prestosql.execution.StageInfo#getAllStages(getStageInfo());
```

Reply:
done